src/meta/expr.lean

```diff
@@ -541,10 +541,15 @@ meta def list_local_const_unique_names (e : expr) : name_set :=
 e.fold mk_name_set
   (λ e' _ es, if e'.is_local_constant then es.insert e'.local_uniq_name else es)
 
-/-- Returns a name_set of all constants in an expression. -/
+/-- Returns a `name_set` of all constants in an expression. -/
 meta def list_constant (e : expr) : name_set :=
 e.fold mk_name_set (λ e' _ es, if e'.is_constant then es.insert e'.const_name else es)
 
+/-- Similar to `list_constant`, but returning a `list name` instead, with the goal of
```

Comment:
Ideally a doc string is readable outside the context where it appears, so describing this in terms of `list_constant` isn't so helpful. Could you expand this (and say what the "right order" is), and note the similar `expr.list_constant` at the end? Note that this is in `meta.expr`, so even if there's an obvious "right order" for `squeeze_simp`, this should be clarified here.